components/table/_table.scss

```diff
@@ -0,0 +1,45 @@
+// Table
+.table {
+
+}
+caption {
+  text-align: left;
+}
+.table__dense {
+  font-size: 81.25%;
+  .table--cell {
+    padding: 0.5rem 0 0.5rem 1rem;
+    &:nth-child(1){
+      padding-left: 0;
+    }
```

Comment:
I think that these can be removed as they are no different from those at line 39.